data/framework/level2.ui

```diff
@@ -103,6 +103,7 @@
                   <object class="GtkLabel">
                     <property name="visible">True</property>
                     <property name="can_focus">False</property>
+                    <property name="margin_top">5</property>
```

Comment:
This is correct, but to be even more consistent, we should apply this to "hover" and "click" options as well.